core/src/io/anuke/mindustry/entities/effect/Lightning.java

```diff
@@ -131,4 +138,16 @@ public void draw(){
     public EntityGroup targetGroup(){
         return bulletGroup;
     }
+
+    protected static boolean hitsInsulator(Lightning l){
+        if (l.lines.size > 1){
+            Vector2 from = (Vector2) l.lines.get(l.lines.size - 2);
```

Comment:
Because else the ide assumes its position object which didn't have a method/property that vector did have that i needed to use, i don't recall it anymore though, will remove the cast 🤔